deepchecks/tabular/checks/distribution/whole_dataset_drift.py

```diff
@@ -93,11 +93,10 @@ def run_logic(self, context: Context) -> CheckResult:
         test_dataset = context.test
         features = train_dataset.features
         cat_features = train_dataset.cat_features
+        numerical_features = train_dataset.numerical_features
 
         sample_size = min(self.sample_size, train_dataset.n_samples, test_dataset.n_samples)
 
-        numerical_features = list(set(features) - set(cat_features))
-
         headnote = """
         <span>
         The shown features are the features that are most important for the domain classifier - the
```

Comment:
Think this would crash, because currently this will try to run on all features rather than numeric+categorical

Reply:
Later it also uses only them, so I don't think it will crash. No reason for it to crash.

Reply:
you are right, sorry didn't look inside.